app/routes/image-set/cv-results.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model: function() {
+    var imageSet = this.modelFor('imageSet');
+    return this.store.find('cvResult', {image_set_id: imageSet.get('id')});
+  },
+
+  setupController: function(controller, model) {
+    var imageSet = this.modelFor('imageSet');
+    controller.setProperties({
+      model: model,
+      searchImageSet: imageSet
```

Comment:
is there perhaps a better variable name to use here? I'm not 100% clear on what the `searchImageSet` is, but from context I think it is the image set the user is "operating on" aka viewing the cv results of?
The name `searchImageSet` makes me think it is the result of some search that the user performed, though?